### DISCUSSION TITLE
With the API doa now might be a good time to ask

### DISCUSSION THREAD

**Discussion by tmabell** [weather-gov/api#689] [Q&A]
Is it commonplace throughout the NWS that WFO's update their API's at times that put them out of sync with each other?  My WFO will update the Forecast API at about 5AM-ish but they don't update the Grids API until as much as an hour later.  This plays havoc with the data as it is displayed on my web page.  Unfortunately we are forced to use both API's because the Forecast API does not contain QPF.

**Comment by sullynole** [comment]
Do you have an example of which URLs are updating at different times?

**Reply by tmabell** [reply]
https://api.weather.gov/gridpoints/IWX/32,63
and
https://api.weather.gov/gridpoints/IWX/32,63/forecast

**Reply by scadergit** [reply]
The office will upload a forecast netcdf that an upstream service (the current cause of the gridpoint outage) will ingest and provide to the API as grids. The API then provides the forecast from the updated grids. Given the forecast data is provided by the grids, any discreptancy could only be due to caching or simliar issues. We don't have reports of this happening overall, so curious what you are experiencing. Once gridpoints are enabled again, it would be helpful to have the correlation id for both requests (from the headers).

**Reply by tmabell** [reply]
This happens routinely during the early morning update when the first period switches from overnight to daytime.  If it is a caching issue it is occurring somewhere external to my systems.  I will provide the headers and cached examples once the API is back online.

**Reply by scadergit** [reply]
Thanks, there is caching at the origin and also at the edge servers of our CDN provider. The correlation ids will help determine if the requests are stale.
